### PR TITLE
Convergence gathering logging

### PR DESCRIPTION
**Pull request by manishtomar**
Logging beginning of convergence cycle and time it takes to gather all the data.

### CODE REVIEW

**Comment by manishtomar** [comment]
jenkins please run integration tests

File: otter/log/spec.py [hunk 1]
```diff
@@ -102,6 +102,7 @@ def length_calc(e):
     # event -> [(event, format_str)]
     "add-server-clb": ("Adding {server_id} with IP address {ip_address} "
                        "to CLB {clb_id}"),
+    "beg-exec-convergence": "Beginning to execute convergence",
```
Comment (radix) [inline otter/log/spec.py]: Just nitpicks: given that we don't do abbreviations so far (plus "beg" kinda sounds like begging ;) I think it'd be better to call this 'begin-convergence', which is just as short

File: otter/log/spec.py [hunk 2]
```diff
@@ -115,6 +116,8 @@ def length_calc(e):
     "execute-convergence": split_execute_convergence,
     "execute-convergence-results": (
         "Got result of {worst_status} after executing convergence"),
+    "get-conv-exec-data": (
```
Comment (radix) [inline otter/log/spec.py]: and another nitpick: I think this could be called "gather-data"

**Comment by radix** [comment]
LGTM! You can ignore my comments if you want :)

**Pull request merged**